### PR TITLE
project module updated

### PR DESCRIPTION
**Pull request by Vibatious**
I have updated the project module

### CODE REVIEW

File: src/store/modules/projects.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	},
+	actions: {
+		projects: firebaseAction(({ commit, bindFirebaseRef}, {ref, callbacks})=>{
+			bindFirebaseRef('projects', ref, callbacks)
+			commit('setProjectsRef', ref)
+		})
+	},
+	getters: {
+		projects:state => state.projects
+		}
```
Comment (iamareebjamal) [inline src/store/modules/projects.js]: The indentation of the file is not correct

File: src/store/modules/projects.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	},
+	actions: {
+		projects: firebaseAction(({ commit, bindFirebaseRef}, {ref, callbacks})=>{
+			bindFirebaseRef('projects', ref, callbacks)
+			commit('setProjectsRef', ref)
+		})
+	},
+	getters: {
+		projects:state => state.projects
```
Comment (iamareebjamal) [inline src/store/modules/projects.js]: Redundant getter

File: src/store/modules/projects.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { firebaseAction } from 'vuexfire'
+import * as firebase from 'firebase'
```
Comment (iamareebjamal) [inline src/store/modules/projects.js]: Unused import

File: src/store/modules/projects.js (hunk 1 above)
Comment (Vibatious) [reply src/store/modules/projects.js]: I will update it

**Pull request closed without merging**